ptf/platform_helper/brcm_t0_sai_helper.py

```diff
@@ -262,6 +265,77 @@ def create_host_intf(self):
                 print("Cannot create hostif, error : {}".format(e))
 
 
+    def config_vlan(self):
+        """
+        Default configuation, with Vlan.
+        Craete bridge_port and route interface
+        Steps:
+        """
+        VLAN_ID = 1000
+        PC_PORT_MTU=9100
+        vlan_id = sai_thrift_create_vlan(self.client, vlan_id=VLAN_ID)
+        for i in range(3, 24):
+            bridge_port_id = sai_thrift_create_bridge_port(self.client,
+                    type=SAI_BRIDGE_PORT_TYPE_PORT,
+                    port_id=self.port_list[i],
+                    admin_state=True,
+                    fdb_learning_mode=SAI_BRIDGE_PORT_FDB_LEARNING_MODE_HW)
+        
+            sai_thrift_set_hostif_attribute(self.client, 
```

Comment:
this method should not belong to creating VLAN, it should be in a method for setting hostifs' attribute